Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,38 @@
+﻿using BootCamp.Chapter;
+using System;
+using System.Collections.Generic;
+
+internal class Lesson3
+{
+    private static List<int> list1 = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+
+    internal static void Demo()
+    {
+        //Demo the Shuffle method
```

Comment:
Each of those demos, instead of a comment, can be a private method. Slightly more cleaner.